products/rhel9/profiles/cis.profile

```diff
@@ -0,0 +1,1084 @@
+documentation_complete: true
+
+metadata:
+    version: 0.0.0
+    SMEs:
+        - vojtapolasek
+        - yuumasato
+
+reference: https://www.cisecurity.org/benchmark/red_hat_linux/
+
+title: '[DRAFT] CIS Red Hat Enterprise Linux 9 Benchmark'
+
+description: |-
+    This is a draft CIS profile based on the RHEL8 CIS
```

Comment:
Small typo in RHEL version, which I think does not block this PR.

Reply:
It is not a typo - it is indeed based on the RHEL8 CIS for the time being.